ui/docs/manual/access-control/user-signin.md

```diff
@@ -0,0 +1,14 @@
+---
+title: User Sign-In
+order: 20
+---
+
+The Taskcluster user interface allows users to sign in, interfacing with various user-authentication systems.
+This implementation is a fairly typical web-application sign-in system, using cookies to maintain sessions.
+
+The user interface backend (taskcluster-web-server) generates Taskcluster Credentials based on the user's identity.
```

Comment:
Should we refer to the backend as taskcluster-web-server or web-server? We should probably pick one.